core/network/subnet.go

```diff
@@ -54,9 +54,14 @@ type SubnetInfo struct {
 	// availability zones.
 	AvailabilityZones []string
 
+	// SpaceID is the id of the space the subnet is associated with.
+	// It can be empty if the subnet is not associated with a space yet.
+	// SpaceID is preferred over SpaceName.
+	SpaceID string
+
 	// SpaceName is the name of the space the subnet is associated with.
 	// It can be empty if the subnet is not associated with a space yet.
-	// TODO (manadart 2019-07-11): Remove in lieu of SpaceId?
+	// OR if the SpaceID is set.
```

Comment:
Under what circumstances is it one versus the other?

Reply:
comments updated